packages/desktop-gui/src/runs/manual-setup.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react'
+import Markdown from 'markdown-it'
+import Tooltip from '@cypress/react-tooltip'
+import { FileOpener } from '../lib/file-opener'
+import appStore from '../lib/app-store'
+import ipc from '../lib/ipc'
+import projectsApi from '../projects/projects-api'
+
+const md = new Markdown({
+  html: true,
+  linkify: true,
+})
+
+function ManualSetup ({ error, configFile, project }) {
+  const relativeFile = configFile
+  const absoluteFile = `${ appStore.projectRoot }/${ relativeFile }`
+  const [isSubmitting, setSubmitting] = useState(false)
+  const [newMessage, setNewMessage] = useState('')
+
+  const codeToToAddKeys = `  projectId: ${JSON.stringify(error.payload.projectId)}`
+
+  const retry = (e) => {
+    e.preventDefault()
+
+    setSubmitting(true)
+
+    projectsApi.reopenProject(project).then((config) => {
+      setSubmitting(false)
+      if (typeof config.projectId !== 'string') {
+        // display a message stating that there was no projectId found
+        setNewMessage('Cypress was not able to resolve a projectId in the config file')
+      }
+    })
+  }
+
+  return (
+    <form onSubmit={retry} className="manual-project-setup">
+      <div className='title-wrapper'>
+        <h4>Cypress can't update your local <code>projectId</code></h4>
+      </div>
+      <p>
+        <span dangerouslySetInnerHTML={{ __html: md.render(error.message) }} />
```

Comment:
Shouldn't it use `<MarkdownRenderer>` here so that external links will be sent through `ipc.openExternal`?